nbgrader/preprocessors/getgrades.py

```diff
@@ -1,9 +1,10 @@
 from .. import utils
 from ..api import Gradebook
+from ..plugins import LateSubmissionPlugin
 from . import NbGraderPreprocessor
 
 
-class GetGrades(NbGraderPreprocessor):
+class GetGrades(NbGraderPreprocessor, LateSubmissionPlugin):
```

Comment:
Also I should note if we do the traitlets thing I mentioned in the main comments, the late submission plugin would then be a trait of the preprocessor, rather than having the preprocessor inherit from it.